src/collections/_documentation/performance/performance-metrics.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Performance Metrics'
+sidebar_order: 2
+---
+
+Sentry's application monitoring product collects performance metrics. Performance metrics make possible a better understanding of how users are interacting with your application. By tracking metrics like Latency and Throughput in [LINK: Discover] or [LINK: Performance], Sentry provides you with a fuller, real-time picture of your application health, such as the occurrence of errors or performance issues.
```

Comment:
```suggestion
Sentry's application monitoring product collects performance metrics. Performance metrics enable you to gain a better understanding of how users are interacting with your application. By tracking metrics like Latency and Throughput in [LINK: Discover] or [LINK: Performance], Sentry provides you with a fuller, real-time picture of your application's health, such as the occurrence of errors or performance issues.
```

Reply:
Trying to avoid the words "enable" and "empower" because they're overused. I switched out the phrasing with "Performance metrics provide a better understanding..."